monitor/src/web-api/web.api.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from "@nestjs/common";
+import { PoolClient } from "pg";
+import { ethers } from "ethers";
```

Comment:
@joseph-bisonai We do not need this `ethers` import.

Reply:
I used this for account balance chacking